pages/linux/toolbox.md

```diff
@@ -0,0 +1,21 @@
+# toolbox
+
+> Tool for containerized command line environments on Linux.
+> Some subcommands such as `create`, `enter`, `init-container`, `list`, `run`, etc. have their own usage documentation, accessible via `tldr toolbox subcommand`.
+> Some distributions require toolbox to run as superuser. More information: <https://github.com/containers/toolbox>.
```

Comment:
This is quite useful to know, but not crucial. And we can only have 3 description lines
```suggestion
> More information: <https://github.com/containers/toolbox>.
```